app/controllers/customers_controller.rb

```diff
@@ -19,8 +21,28 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @customer.update(customer_params)
+      redirect_to customers_path
```

Comment:
acho que o ideal seria redirecionar para o show do customer, imagina que vc acaba de atualizar o cadastro de alguem e ai o sistema te manda de volta pra tela de listagem, com ctz vc vai ter que acessar de novo o cadastro do cara pra validar se realmente foi alterado, então seria melhor redirecionar o cara pra o show do customer :wink: 
```suggestion
      redirect_to @customer
```